test/test-knative.js

```diff
@@ -85,7 +85,7 @@ describe('cloud-assets:knative', function () {
 		  }
 
 		let generatedYamlContents = yml.safeLoad(fs.readFileSync(serviceYamlFilePath, 'utf8'));
-		assert(_.isEqual(generatedYamlContents, targetServiceYaml));
+		assert.textEqual(JSON.stringify(generatedYamlContents, null, 3), JSON.stringify(targetServiceYaml, null, 3));
```

Comment:
I dont think stringify will work here as it will fail for unordered keys

Reply:
I just changed it to make debugging easier. When the tests were failing it was printing out without issues.